### DISCUSSION TITLE
Beta for version 5

### DISCUSSION THREAD

**Discussion by REMC19** [Aries85/LightZone#293] [General]
Hi Masahiro
I see that you have now released a beta version 5  of Lightzone.
Would you like me to make mention of it on Pixls, or would you prefer to do that yourself?
I did notice that one or two issue had already been raised regarding the beta - would you prefer for them to have been resolved before making the availability of the beta version 5 more widely know?
One more question - can the beta version 5 be run along side an existing version 4 installation of the program - or does it replace it.
Thanks once again for your efforts.

Edit - oops I see that you already have, I have just scrolled down far enough on the Pixls.us discussion page to see it!

**Comment by ktgw0316** [comment]
Hi,

> can the beta version 5 be run along side an existing version 4 installation of the program - or does it replace it.

You can keep the v4 along with v5 on Windows if you select "No, install into a different directory" during installation. On Linux, it depends on your distribution.
